module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -125,7 +125,7 @@ public function getAvailableDigitalFormats()
                     $formatType = $format->formatType;
                     $formats[$formatType] = $formatNames[$formatType];
                 }
-            // If we aren't locked in, we can show all formats
+                // If we aren't locked in, we can show all formats
```

Comment:
I think it would be cleaner to rephrase this so that it can be moved into the next `else` block, like `// Not locked in, show all formats`.